src/sentry/utils/performance_issues/detectors/io_main_thread_detector.py

```diff
@@ -122,10 +121,6 @@ class FileIOMainThreadDetector(BaseIOMainThreadDetector):
     settings_key = DetectorType.FILE_IO_MAIN_THREAD
     group_type = PerformanceFileIOMainThreadGroupType
 
-    @classmethod
-    def is_detector_enabled(cls) -> bool:
-        return not options.get("performance_issues.file_io_main_thread.disabled")
```

Comment:
Nothing should be changed ingest-wise with this change because this flag was defaulted to `False` meaning this would enable the detector in every region. It is not overridden anywhere to no worries there.

Using the detector flag `performance.issues.file_io_main_thread.problem-creation` instead will also work just fine because it was already being later in the pipeline anyway.